Ошибка при импорте отчета Тинькофф

**Опишите ошибку**
При импорте Тинькофф приложение падает
```
2024-05-13 06:46:45 Setting Active Processor Count to 6
2024-05-13 06:46:45 Calculating JVM memory based on 10366664K available memory
2024-05-13 06:46:45 For more information on this calculation, see https://paketo.io/docs/reference/java-reference/#memory-calculator
2024-05-13 06:46:45 Calculated JVM Memory Configuration: -XX:MaxDirectMemorySize=10M -Xmx9713103K -XX:MaxMetaspaceSize=141560K -XX:ReservedCodeCacheSize=240M -Xss1M (Total Memory: 10366664K, Thread Count: 250, Loaded Class Count: 22579, Headroom: 0%)
2024-05-13 06:46:45 Enabling Java Native Memory Tracking
2024-05-13 06:46:45 Spring Cloud Bindings Enabled
2024-05-13 06:46:47 
2024-05-13 06:46:47   .   ____          _            __ _ _
2024-05-13 06:46:47  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
2024-05-13 06:46:47 ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
2024-05-13 06:46:47  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
2024-05-13 06:46:47   '  |____| .__|_| |_|_| |_\__, | / / / /
2024-05-13 06:46:47  =========|_|==============|___/=/_/_/_/
2024-05-13 06:46:47  :: Spring Boot ::                (v3.0.7)
2024-05-13 06:46:47 
2024-05-13 06:46:47 2024-05-13T03:46:47.178Z  INFO 1 --- [           main] ru.investbook.InvestbookApplication      : Starting InvestbookApplication using Java 21.0.1 with PID 1 (/workspace/BOOT-INF/classes started by cnb in /workspace)
2024-05-13 06:46:47 2024-05-13T03:46:47.183Z  INFO 1 --- [           main] ru.investbook.InvestbookApplication      : The following 3 profiles are active: "core", "h2", "conf"
2024-05-13 06:46:49 2024-05-13T03:46:49.699Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-05-13 06:46:49 2024-05-13T03:46:49.914Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 205 ms. Found 14 JPA repository interfaces.
2024-05-13 06:46:50 2024-05-13T03:46:50.977Z  WARN 1 --- [           main] io.undertow.websockets.jsr               : UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2024-05-13 06:46:50 2024-05-13T03:46:50.996Z  INFO 1 --- [           main] io.undertow.servlet                      : Initializing Spring embedded WebApplicationContext
2024-05-13 06:46:50 2024-05-13T03:46:50.996Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 3704 ms
2024-05-13 06:46:51 2024-05-13T03:46:51.184Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2024-05-13 06:46:51 2024-05-13T03:46:51.560Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:file:~/investbook/investbook2 user=SA
2024-05-13 06:46:51 2024-05-13T03:46:51.570Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2024-05-13 06:46:51 2024-05-13T03:46:51.614Z  INFO 1 --- [           main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:file:~/investbook/investbook2'
2024-05-13 06:46:52 2024-05-13T03:46:52.082Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : Flyway Community Edition 9.5.1 by Redgate
2024-05-13 06:46:52 2024-05-13T03:46:52.083Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : See what's new here: https://flywaydb.org/documentation/learnmore/releaseNotes#9.5.1
2024-05-13 06:46:52 2024-05-13T03:46:52.083Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : 
2024-05-13 06:46:52 2024-05-13T03:46:52.111Z  INFO 1 --- [           main] o.f.c.i.database.base.BaseDatabaseType   : Database: jdbc:h2:file:~/investbook/investbook2 (H2 2.1)
2024-05-13 06:46:52 2024-05-13T03:46:52.184Z  INFO 1 --- [           main] o.f.c.i.c.SqlScriptCallbackFactory       : Executing SQL callback: beforeValidate - 
2024-05-13 06:46:52 2024-05-13T03:46:52.245Z  INFO 1 --- [           main] o.f.core.internal.command.DbValidate     : Successfully validated 26 migrations (execution time 00:00.030s)
2024-05-13 06:46:52 2024-05-13T03:46:52.252Z  INFO 1 --- [           main] o.f.c.i.s.JdbcTableSchemaHistory         : Creating Schema History table "PUBLIC"."flyway_schema_history" ...
2024-05-13 06:46:52 2024-05-13T03:46:52.279Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "PUBLIC": << Empty Schema >>
2024-05-13 06:46:52 2024-05-13T03:46:52.313Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "1 - init schema"
2024-05-13 06:46:52 2024-05-13T03:46:52.411Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2020.7.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.427Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2020.11.0.0"
2024-05-13 06:46:45 Picked up JAVA_TOOL_OPTIONS: -Djava.security.properties=/layers/paketo-buildpacks_bellsoft-liberica/java-security-properties/java-security.properties -XX:+ExitOnOutOfMemoryError -XX:ActiveProcessorCount=6 -XX:MaxDirectMemorySize=10M -Xmx9713103K -XX:MaxMetaspaceSize=141560K -XX:ReservedCodeCacheSize=240M -Xss1M -XX:+UnlockDiagnosticVMOptions -XX:NativeMemoryTracking=summary -XX:+PrintNMTStatistics -Dorg.springframework.cloud.bindings.boot.enable=true
2024-05-13 06:46:52 2024-05-13T03:46:52.448Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2020.13.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.461Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2020.14.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.471Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.1.1.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.491Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.2.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.523Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.2.1.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.536Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.4.1.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.541Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.5.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.551Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.6.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.559Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.8.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.591Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.9.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.661Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.762Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.1"
2024-05-13 06:46:52 2024-05-13T03:46:52.788Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.2"
2024-05-13 06:46:52 2024-05-13T03:46:52.800Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.3"
2024-05-13 06:46:52 2024-05-13T03:46:52.884Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.4"
2024-05-13 06:46:52 2024-05-13T03:46:52.896Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2021.11.0.5"
2024-05-13 06:46:52 2024-05-13T03:46:52.912Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2022.1.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.922Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2022.1.0.1"
2024-05-13 06:46:52 2024-05-13T03:46:52.931Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2022.2.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.951Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2022.3.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.961Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2023.1.0.0"
2024-05-13 06:46:52 2024-05-13T03:46:52.972Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2023.2.0.0"
2024-05-13 06:46:53 2024-05-13T03:46:53.000Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version "2023.4.0.0"
2024-05-13 06:46:53 2024-05-13T03:46:53.010Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Successfully applied 26 migrations to schema "PUBLIC", now at version v2023.4.0.0 (execution time 00:00.733s)
2024-05-13 06:46:53 2024-05-13T03:46:53.189Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2024-05-13 06:46:53 2024-05-13T03:46:53.236Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.1.7.Final
2024-05-13 06:46:53 2024-05-13T03:46:53.487Z  INFO 1 --- [           main] SQL dialect                              : HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
2024-05-13 06:46:54 2024-05-13T03:46:54.758Z  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2024-05-13 06:46:54 2024-05-13T03:46:54.785Z  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2024-05-13 06:46:56 2024-05-13T03:46:56.987Z  INFO 1 --- [           main] r.i.parser.TableFactoryRegistryService   : 0 extensions implementations of table factory found in PT0.000549084S: []
2024-05-13 06:46:57 2024-05-13T03:46:57.878Z  INFO 1 --- [           main] o.s.b.a.w.s.WelcomePageHandlerMapping    : Adding welcome page template: index
2024-05-13 06:46:58 2024-05-13T03:46:58.449Z  INFO 1 --- [           main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 3 endpoint(s) beneath base path '/actuator'
2024-05-13 06:46:58 2024-05-13T03:46:58.544Z  INFO 1 --- [           main] io.undertow                              : starting server: Undertow - 2.3.6.Final
2024-05-13 06:46:58 2024-05-13T03:46:58.554Z  INFO 1 --- [           main] org.xnio                                 : XNIO version 3.8.8.Final
2024-05-13 06:46:58 2024-05-13T03:46:58.562Z  INFO 1 --- [           main] org.xnio.nio                             : XNIO NIO Implementation Version 3.8.8.Final
2024-05-13 06:46:59 2024-05-13T03:46:59.039Z  INFO 1 --- [           main] org.jboss.threads                        : JBoss Threads version 3.5.0.Final
2024-05-13 06:46:59 2024-05-13T03:46:59.127Z  INFO 1 --- [           main] o.s.b.w.e.undertow.UndertowWebServer     : Undertow started on port(s) 2030 (http)
2024-05-13 06:46:59 2024-05-13T03:46:59.199Z  INFO 1 --- [           main] ru.investbook.InvestbookApplication      : Started InvestbookApplication in 12.676 seconds (process running for 13.722)
2024-05-13 06:46:59 2024-05-13T03:46:59.633Z  INFO 1 --- [  XNIO-1 task-2] io.undertow.servlet                      : Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-05-13 06:46:59 2024-05-13T03:46:59.633Z  INFO 1 --- [  XNIO-1 task-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2024-05-13 06:46:59 2024-05-13T03:46:59.637Z  INFO 1 --- [  XNIO-1 task-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 3 ms
2024-05-13 06:47:05 2024-05-13T03:47:05.935Z ERROR 1 --- [  XNIO-1 task-4] io.undertow.request                      : UT005023: Exception handling request to /broker-file-reports
2024-05-13 06:47:05 
2024-05-13 06:47:05 jakarta.servlet.ServletException: Handler dispatch failed: java.lang.NoSuchMethodError: 'org.apache.commons.io.input.UnsynchronizedByteArrayInputStream$Builder org.apache.commons.io.input.UnsynchronizedByteArrayInputStream.builder()'
2024-05-13 06:47:05 at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1096) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:974) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1011) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:914) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:547) ~[jakarta.servlet-api-6.0.0.jar:6.0.0]
2024-05-13 06:47:05 at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614) ~[jakarta.servlet-api-6.0.0.jar:6.0.0]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:129) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at org.springframework.web.filter.ServerHttpObservationFilter.doFilterInternal(ServerHttpObservationFilter.java:109) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:117) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.SendErrorPageHandler.handleRequest(SendErrorPageHandler.java:52) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:276) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:132) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:256) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:101) ~[undertow-servlet-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.server.Connectors.executeRootHandler(Connectors.java:393) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:859) ~[undertow-core-2.3.6.Final.jar:2.3.6.Final]
2024-05-13 06:47:05 at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18) ~[jboss-threads-3.5.0.Final.jar:3.5.0.Final]
2024-05-13 06:47:05 at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513) ~[jboss-threads-3.5.0.Final.jar:3.5.0.Final]
2024-05-13 06:47:05 at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538) ~[jboss-threads-3.5.0.Final.jar:3.5.0.Final]
2024-05-13 06:47:05 at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282) ~[xnio-api-3.8.8.Final.jar:3.8.8.Final]
2024-05-13 06:47:05 at java.base/java.lang.Thread.run(Unknown Source) ~[na:na]
2024-05-13 06:47:05 Caused by: java.lang.NoSuchMethodError: 'org.apache.commons.io.input.UnsynchronizedByteArrayInputStream$Builder org.apache.commons.io.input.UnsynchronizedByteArrayInputStream.builder()'
2024-05-13 06:47:05 at org.apache.poi.openxml4j.util.ZipArchiveFakeEntry.getInputStream(ZipArchiveFakeEntry.java:108) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.getInputStream(ZipInputStreamZipEntrySource.java:112) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.openxml4j.opc.ZipPackage.getPartsImpl(ZipPackage.java:305) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.openxml4j.opc.OPCPackage.getParts(OPCPackage.java:781) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:354) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:64) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:313) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:287) ~[poi-ooxml-5.2.5.jar:5.2.5]
2024-05-13 06:47:05 at ru.investbook.parser.AbstractExcelBrokerReport.getWorkBook(AbstractExcelBrokerReport.java:41) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.tinkoff.TinkoffBrokerReport.<init>(TinkoffBrokerReport.java:53) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.tinkoff.TinkoffBrokerReportFactory.lambda$create$0(TinkoffBrokerReportFactory.java:55) ~[classes/:na]
2024-05-13 06:47:05 at org.spacious_team.broker.report_parser.api.AbstractBrokerReportFactory.create(AbstractBrokerReportFactory.java:57) ~[broker-report-parser-api-2024.1.jar:na]
2024-05-13 06:47:05 at ru.investbook.parser.tinkoff.TinkoffBrokerReportFactory.create(TinkoffBrokerReportFactory.java:54) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.BrokerReportParserServiceImpl.getReportOfUnknownBroker(BrokerReportParserServiceImpl.java:144) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.BrokerReportParserServiceImpl.getBrokerReport(BrokerReportParserServiceImpl.java:133) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.BrokerReportParserServiceImpl.parseReport0(BrokerReportParserServiceImpl.java:95) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.parser.BrokerReportParserServiceImpl.parseReport(BrokerReportParserServiceImpl.java:64) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.web.BrokerFileReportController.uploadReport(BrokerFileReportController.java:81) ~[classes/:na]
2024-05-13 06:47:05 at ru.investbook.web.BrokerFileReportController.lambda$uploadBrokerReports$1(BrokerFileReportController.java:70) ~[classes/:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ForEachOps$ForEachTask.compute(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.concurrent.CountedCompleter.exec(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.concurrent.ForkJoinTask.invoke(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at ru.investbook.web.BrokerFileReportController.uploadBrokerReports(BrokerFileReportController.java:69) ~[classes/:na]
2024-05-13 06:47:05 at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
2024-05-13 06:47:05 at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:207) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:152) ~[spring-web-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:884) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:797) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1081) ~[spring-webmvc-6.0.9.jar:6.0.9]
2024-05-13 06:47:05 ... 54 common frames omitted
2024-05-13 06:47:05 
```

**Последовательность действий для воспроизведения**
Шаги по воспроизведению ошибки:
Выбрать страницу импорта
Выбрать отчёт Тинькофф
Нажать загрузить
Ошибка


**Ожидаемый результат**
на версии `2023.4` Данной ошибки нету

**Скриншоты**
Если возможно, добавьте скриншоты, чтобы объяснить проблему.

**Окружение:**
 - mac (docker run)
 - chrome
 - 2024.1
 - https://hub.docker.com/layers/spaciousteam/investbook/2024.1.1/images/sha256-3f55dfc94c953905c58b6460c7b14566cdb6b0d7553e20c1585e7077b511e952?context=explore

**Дополнительный контекст**
Добавьте сюда любой другой контекст проблемы.

--- a/pom.xml
+++ b/pom.xml
@@ -101,7 +101,7 @@
         <dependency>
             <groupId>com.github.spacious-team</groupId>
             <artifactId>table-wrapper-spring-boot-starter</artifactId>
-            <version>2024.1</version>
+            <version>2024.1</version>  <!-- if changed, update the version of poi-scratchpad -->
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
@@ -220,7 +220,7 @@
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi-scratchpad</artifactId>
-            <version>5.2.3</version>  <!-- use same version as table-wrapper-spring-boot-starter provides -->
+            <version>5.2.5</version>  <!-- use same version as table-wrapper-spring-boot-starter provides -->
         </dependency>
     </dependencies>
 

